modules/ROOT/pages/acm-lightning-components.adoc

```diff
@@ -68,37 +68,34 @@ Filter Condition (Advanced):: Causes the carousel to show only APIs that match t
 Learn More Button Label:: Enables you to set the Learn More button label. Default is *Learn More*.
 
-=== Filter Conditions [[filter-conditions]]
+== API Catalog
 
-The API Carousel component parameter *Filter Condition (Advanced)* causes the carousel to show only the APIs that meet the condition. A condition is a standard search in Salesforce Object Query Language (SOQL).
+The API Catalog offers API consumers an interactive way to explore all the available APIs in an API community.
 
-You can search these fields:
+The API Catalog component shows API cards in rows and columns, divided into pages, and controls to filter the displayed APIs by categories and search terms. The catalog is suitable for displaying and exploring large numbers of API versions.
 
-* `+acm_pkg__Icon__c+`
-* `+acm_pkg__Name__c+`
-* `+acm_pkg__NavigateUrl__c+`
-* `+acm_pkg__VersionGroup__c+`
+In the template community, the API catalog link appears below the API carousel. If a community has multiple API catalog instances on multiple pages, the best practice is to standardize the pages and place the instances as consistently as possible.
 
-These examples demonstrate filter conditions:
+To prevent confusion, the API catalog is designed to use managed categories and not tags, and limit the number of categories to five.
 
-* The API name is exactly equal to `My API`: `+acm_pkg__Name__c = 'My API'+`
-* The API name contains the text `My API`: `+acm_pkg__Name__c LIKE '%My API%'+`
-* The API name is `API Alpha` or `API Bravo`: `+(acm_pkg__Name__c = 'API Alpha' OR acm_pkg__Name__c = 'API Bravo')+`
-* The API name is `My API` and the API version group is `v1`: `+(acm_pkg__Name__c = 'My API' AND acm_pkg__VersionGroup__c = 'v1')+`
-* The API name begins with `Product`, `Shipping`, or `Order`: `+(acm_pkg__Name__c LIKE 'Product%' OR acm_pkg__Name__c LIKE 'Shipping%' OR acm_pkg__Name__c LIKE 'Order%')+`
-* The API has an icon: `+acm_pkg__Icon__c != ''+`
-* The API has a Learn More button that links to a custom page with more information: `+acm_pkg__NavigateUrl__c != ''+`
-* The API version group is exactly equal to `v2`: `+acm_pkg__VersionGroup__c = 'v2'+`
-* The API version group is later than `v1`: `+acm_pkg__VersionGroup__c > 'v1'+`
-* The API version group is earlier than `v1`: `+acm_pkg__VersionGroup__c < 'v1'+`
-* The API version group is later than or equal to `v2`: `+acm_pkg__VersionGroup__c >= 'v2'+`
-* The API version group is earlier than or equal to `v2`: `+acm_pkg__VersionGroup__c <= 'v2'+`
+A card may have multiple badges from one category. A card displays up to three badges, and if there are more, offers a tooltip to display them.
 
-Queries using `OR` or `AND` must enclose the whole condition in parentheses.
+If category colors are not specified, the community theme is used for defaults. Colors can be hexadecimal values such as `#FF0000` or `#0000FF`, or standard CSS color names such as `red` and `blue`.
```

Comment:
Are we using passive voice intentionally, to refer to what users other than our audience experience? If not...:
```suggestion
If you do not specify category colors, your community theme is used as the default. Colors can be hexadecimal values such as `#FF0000` or `#0000FF` or standard CSS color names such as `red` and `blue`.
```

Reply:
That was accidental, and this is much better. Thank you!